src/common-utils/src/data-type.ts

```diff
@@ -0,0 +1,255 @@
+// SPDX-License-Identifier: MIT
+// Copyright contributors to the kepler.gl project
+
+import {Analyzer, DATA_TYPES as AnalyzerDATA_TYPES} from 'type-analyzer';
+import {RowData, Field} from '@kepler.gl/types';
+import {ALL_FIELD_TYPES} from '@kepler.gl/constants';
+import {console as globalConsole} from 'global/window';
+import {range} from 'd3-array';
+import {isHexWkb, notNullorUndefined} from './data';
+
+export const ACCEPTED_ANALYZER_TYPES = [
```

Comment:
Nit: Can we add a TSDoc comment in front of every type, function, field/member/method?

In this comment maybe state what it would mean for a type not to be accepted.